batch/batch/cloud/azure/driver/create_instance.py

```diff
@@ -190,6 +190,7 @@ def create_vm_config(
 DOCKER_PREFIX=$(jq -r '.docker_prefix' userdata)
 
 INTERNAL_GATEWAY_IP=$(jq -r '.internal_ip' userdata)
+REGION=$(jq -r '.region' userdata)
```

Comment:
Probably better to grab this from the instance config more like how we do in GCP. I could see this getting missed if we added more regions to azure.